### PR TITLE
implement NOTARGET cheat

### LINKED ISSUE #681: NOTARGET cheat?
Self explanatory, only suggesting because it's really nice to have for wad testing purposes.

### PR DESCRIPTION
**Pull request by rfomin**
Fix #681

### CODE REVIEW

File: src/p_enemy.c [hunk 1]
```diff
@@ -1035,14 +1043,20 @@ void A_Look(mobj_t *actor)
 {
   mobj_t *targ;
 
+  targ = actor->subsector->sector->soundtarget;
```
Comment (fabiangreffrath) [inline src/p_enemy.c:1046]: Can this be const?

Comment (rfomin) [reply src/p_enemy.c:1046]: It is later used in `P_SetTarget` and `P_CheckSight`, so we need to change the parameter declarations for these functions.

Comment (fabiangreffrath) [reply src/p_enemy.c:1046]: Then it's okay.

**Review by fabiangreffrath — APPROVED** [review]

**Comment by kraflab** [comment]
Amazing continued progress on woof this year 👏

**Comment by fabiangreffrath** [comment]
> Amazing continued progress on woof this year 👏

@rfomin goes full steam ahead. 😁

**Pull request merged**